changes/bug23576

```diff
@@ -0,0 +1,7 @@
+  o Minor features (IPv6, v3 onion services):
+    - Onion services put IPv6 addresses in service descriptors.
```

Comment:
Nitpick: Perhaps the commit msg here should be `Onion services put IPv6 addresses in service descriptors.` instead of `Rewrite service_intro_point_new() to take a node_t`.

Reply:
See my branch bug23576-rebased, where I rebased and changed this commit summary.